frontend/app/NearlineUnclog.jsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import axios from 'axios';
+import {Pie,Bar,HorizontalBar} from "react-chartjs-2";
+import NearlineControlsBanner from "./common/NearlineControlsBanner.jsx";
+import ProjectSearchView from "./ProjectSearchView.jsx";
+import BytesFormatterImplementation from "./common/BytesFormatterImplementation.jsx";
+
+class NearlineUnclog extends React.Component {
+    constructor(props){
+        super(props);
+
+        this.state = {
+            loading: false,
+            lastError:null,
+            chartMode: NearlineControlsBanner.CHART_MODE_COUNT,
+            allData: null,
+            showProjectsList: false,
+            selectedProjectStatus: null
+        };
+
+        this.refresh = this.refresh.bind(this);
+    }
+
+    static makeColourValues(count, offset){
+        let values = [];
+        for(let n=0;n<count;++n){
+            let hue = (n/count)*360.0 + offset;
+            values[n] = 'hsla(' + hue + ',75%,50%,0.6)'
+        }
+        return values;
+    }
+
+    static colourValues = NearlineUnclog.makeColourValues(19, 10);
+
+    static makeHoverColourValues(count, offset){
```

Comment:
personally i would just add a parameter to `makeColourValues` for the alpha value, and then call it once with 0.6 for the `colourValues` and once with 0.8 for the `hoverColourValues`

Reply:
Done.